datahub-graphql-core/src/main/resources/gms.graphql

```diff
@@ -1140,17 +1230,33 @@ type CorpGroup implements Entity {
     type: EntityType!
 
     """
-    group name e.g. wherehows-dev, ask_metadata
+    URL-encoded group name e.g. wherehows-dev, ask_metadata
```

Comment:
why url encode when sending to the client? I would think we'd only want to url encode right before putting into a url (or after retrieving)

Reply:
Yeah thats a good point. I don't think we need it to be - you're right